patches/chromium/fix_setparentacessibile_crash_win.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Biru Mohanathas <birunthan@mohanathas.com>
+Date: Thu, 10 Dec 2020 19:02:37 +0200
+Subject: fix crash in NativeViewHost::SetParentAccessible
+
+This fixes random crashes on Windows 10. It presumably started happening
+after the changes in
+https://chromium.googlesource.com/chromium/src.git/+/5c6c8e994bce2bfb867279ae5068e9f9134e70c3%5E!/#F15
+
+For context, see: https://github.com/electron/electron/issues/26905
```

Comment:
We require all patches to describe a plan for their own removal, or else a reason why they should be maintained forever, per https://github.com/electron/electron/blob/master/docs/development/patches.md#patch-justification.

Can this patch be upstreamed? If not, why not? Can we fix this crash without a patch?

Reply:
@nornagon Updated!